classes/external/get_kanban_content.php

```diff
@@ -643,7 +643,7 @@ public static function get_discussion_update(int $cmid, int $boardid, int $cardi
 
         $formatter = new updateformatter();
         foreach ($discussions as $discussion) {
-            $discussion->content = format_text($discussion->content);
+            $discussion->content = format_text($discussion->content, true, ['escape' => false]);
```

Comment:
```suggestion
            $discussion->content = format_text($discussion->content, FORMAT_HTML, ['escape' => false]);
```

Reply:
format_text() expects a format as second parameter

Reply:
Yes, good catch, thanks, I got confused with the documentation of `format_string` instead of `format_text` here!

BTW, why it worked anyway: `FORMAT_HTML` is defined as 1 and apparently true has been evaluated to 1, so technically I have not been completely wrong there ;-)